Doppler.MercadoPagoApi/Services/MercadoPagoService.cs

```diff
@@ -0,0 +1,61 @@
+using Doppler.MercadoPagoApi.Models;
+using MercadoPago.Client;
+using MercadoPago.Client.Customer;
+using MercadoPago.Config;
+using MercadoPago.Resource;
+using MercadoPago.Resource.CardToken;
+using MercadoPago.Resource.Customer;
+using Microsoft.Extensions.Configuration;
+using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Net.Http.Json;
+using System.Threading.Tasks;
+
+namespace Doppler.MercadoPagoApi.Services
+{
+    public class MercadoPagoService : IMercadoPagoService
+    {
+        private readonly IConfiguration _configuration;
+        private readonly IMercadoPagoClient _mpClient;
+
+        public MercadoPagoService(IConfiguration configuration, IMercadoPagoClient mpClient)
+        {
+            _configuration = configuration;
+            MercadoPagoConfig.AccessToken = _configuration["MercadoPago:AccessToken"];
```

Comment:
What is `MercadoPagoConfig`? This code looks weird.

Reply:
In that case, it is not a good place to set up it because you are setting up a singleton value when you create a service.

Maybe startup.cs is a better place.